blog-migration.js

```diff
@@ -239,7 +266,7 @@ async function handleMigration(markdown, entry, pageIndex, outputDir) {
         pageReports.push(...pathReports);
     }
 
-    return pageReports;
+    return [ ...pageReports, totalLinksReport ];
```

Comment:
This probably shouldn't be destructured at this stage

Reply:
This was the only way this return patter did work. Destructuring below where it was threw errors in specific instances. If you think is is a problem, we can amend the way links report gets folded in.